packages/babel-standalone/package.json

```diff
@@ -5,8 +5,7 @@
   "main": "babel.js",
   "files": [
     "babel.js",
-    "babel.min.js",
-    "src"
```

Comment:
I remove the `src` here because
1) It is untranspiled since day 1 of `@babel/standalone` and nobody ever complains about that
2) `src` is not intended to be used by end users